stdlib/weakref.pyi

```diff
@@ -65,6 +65,10 @@ class WeakValueDictionary(MutableMapping[_KT, _VT]):
     def copy(self) -> WeakValueDictionary[_KT, _VT]: ...
     __copy__ = copy
     def __deepcopy__(self, memo: Any) -> Self: ...
+    @overload
+    def get(self, key: _KT, default: None = None) -> _VT | None: ...
+    @overload
+    def get(self, key: _KT, default: _KT | _T) -> _VT | _T: ...
```

Comment:
Shouldn't this be `_VT | _T` too? Assuming it shouldn't actually be just `_T` as pointed out in the previous comment.

Reply:
whoooops, good catch!

```suggestion
    def get(self, key: _KT, default: _VT | _T) -> _VT | _T: ...
```